packages/vue/src/components/atoms/SfBullets/SfBullets.js

```diff
@@ -6,14 +6,14 @@ export default {
      */
     total: {
       type: Number,
-      default: 0
+      default: 3
```

Comment:
```suggestion
      default: 1
```

Why 3 by default? I would set 1 or 0...

Reply:
see below